tools/run_all_tests.sh

```diff
@@ -132,16 +122,24 @@ execute_test() {
 
     case $test_scope in
     "all")
+        make all_tests
+        make integration_tests
+        make timing_integrationtests
+        ;;
+    "no_timing_test")
         make all_tests
         ;;
     "unit")
         make module_tests
+        make timing_moduletests
```

Comment:
this is a bit inconsistent. `mudule_tests` is separated with an underscore but not in `timing_moduletests`